padertorch/ops/mappings.py

```diff
@@ -20,29 +43,32 @@
     identity=torch.nn.Sequential,  # https://github.com/pytorch/pytorch/issues/9160
 )
 
-POOLING_FN_MAP = Dispatcher(
-    median=np.median,
-    average=np.mean,
-    min=np.min,
-    max=np.max,
-)
 
-TORCH_POOLING_FN_MAP = Dispatcher(
-    median=torch.median,
-    average=torch.mean,
-    min=torch.min,
-    max=torch.max,
-)
-
-DTYPE_MAP = Dispatcher(
-    float32=np.float32,
-    float64=np.float64,
-    complex64=np.complex64,
-    complex128=np.complex128,
-)
-
-OPTIMIZER_MAP = Dispatcher(
-    sgd=optim.SGD,
-    adam=optim.Adam,
-    adagrad=optim.Adagrad
-)
+# These mappings are not used at the moment if required they can be added again
+# but the naming convention shuld be updated.
+# POOLING_FN_MAP = CallableDispatcher(
```

Comment:
change to `REDUCE_MAP`

Reply:
Sorry, `NP_REDUCE_MAP`

Reply:
I have not updated the names since the comments states that the names are not up to date

Reply:
done

Reply:
Ups, I didn't read the comment.